### PR TITLE
feat: add readiness and liveness probe

### PR DESCRIPTION
**Pull request by hzliangbin**
Signed-off-by: hzliangbin <hzliangbin@foxmail.com>

ref: https://github.com/cectc/dbpack/issues/52

### Ⅰ. Describe what this PR did
add readiness and liveness probe

### Ⅱ. Does this pull request fix one issue?
 #52

### CODE REVIEW

File: cmd/healthcheck.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package main
```
Comment (dk-lockdown) [inline cmd/healthcheck.go:17]: pls add license header at front of this file

**Review by dk-lockdown — APPROVED** [review]

File: cmd/metrics.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package main
```
Comment (dk-lockdown) [inline cmd/metrics.go:17]: license header too

File: cmd/routes.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package main
```
Comment (dk-lockdown) [inline cmd/routes.go:17]: license header.

**Comment by hzliangbin** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by dk-lockdown — APPROVED** [review]

File: cmd/healthcheck.go [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+const (
+	healthCheckLivenessPath  = "/live"
+	healthCheckReadinessPath = "/ready"
+)
+
+func registerHealthCheckRouter(router *mux.Router) {
+	router.Methods(http.MethodGet).Path(healthCheckReadinessPath).HandlerFunc(readinessHandler)
+
+	router.Methods(http.MethodGet).Path(healthCheckLivenessPath).HandlerFunc(livenessHandler)
+
```
Comment (dk-lockdown) [inline cmd/healthcheck.go]: this blank line is unnecessary

File: cmd/routes.go [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+package main
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func configServerHandler() (http.Handler, error) {
```
Comment (dk-lockdown) [inline cmd/routes.go]: suggest change func name to `registerRoutes`

File: cmd/metrics.go [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+import (
+	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// registerMetricsRouter - add handler functions for metrics.
+func registerMetricsRouter(router *mux.Router) {
+	// metrics router
+	router.Path("/metrics").Handler(promhttp.Handler())
+
```
Comment (dk-lockdown) [inline cmd/metrics.go]: this blank line is unnecessary

**Comment by fatelei** [comment]
some comment not resolve

**Comment by adofsauron** [comment]
LGTM

**Review by Lvnszn — APPROVED** [review]

**Comment by fatelei** [comment]
LGTM

**Pull request merged**